bigip/resource_bigip_ltm_monitor.go

```diff
@@ -310,7 +316,7 @@ func resourceBigipLtmMonitorDelete(d *schema.ResourceData, meta interface{}) err
 
 func validateParent(v interface{}, k string) ([]string, []error) {
 	p := v.(string)
-	if p == "/Common/http" || p == "/Common/https" || p == "/Common/icmp" || p == "/Common/gateway-icmp" || p == "/Common/tcp" || p == "/Common/tcp-half-open" || p == "/Common/ftp" {
+	if p == "/Common/udp" || p == "/Common/postgresql" || p == "/Common/http" || p == "/Common/https" || p == "/Common/icmp" || p == "/Common/gateway-icmp" || p == "/Common/tcp" || p == "/Common/tcp-half-open" || p == "/Common/ftp" {
```

Comment:
how about creating a const hash with these values and check against that? seems like we're abusing `if` here.

Reply:
created monitors map and checking membership in it